gcc/testsuite/gcc.target/riscv/cv-mac-mac.c

```diff
@@ -0,0 +1,34 @@
+/* { dg-do compile } */
```

Comment:
Unless I'm mistaken on dejagnu directives, this testcase is going to compile but not run, so the `abort`s won't ever trigger. 
```suggestion
/* { dg-do run } */
```

Reply:
Thank you for pointing this out. I've updated the commit with your changes.

Reply:
Updated